src/web/index.js

```diff
@@ -144,7 +144,8 @@ render(
       clipboard,
       storage,
       isMobile,
-      isFirefox
+      isFirefox,
+      debug: DEBUG
```

Comment:
Don't really need to pass this as a prop, IMO - just import DEBUG in the components and use it directly. It's a build-time thing, which webpack can actually exclude from the bundled JS with tree-shaking if DEBUG is false. Not a hugebig deal though